runtime/kusama/src/lib.rs

```diff
@@ -2405,7 +2405,11 @@ mod init_state_migration {
 			if MigrationProcess::<Runtime>::get() == Default::default() &&
 				AutoLimits::<Runtime>::get().is_none()
 			{
-				AutoLimits::<Runtime>::put(Some(MigrationLimits { item: 160, size: 204800 }));
+				// We use limits to target 600ko proofs per block and
+				// avg 800_000_000_000 of weight per block.
```

Comment:
800 seconds?

Reply:
800_000_000_000 is 800ms I believe

Reply:
Can lower to 600 (using 3200 instead of 4800 move to 510 blocks the migration time so not taking really longer).

Could make sense to calculate better weight as I believe it is an overestimate (iterating on data since iterator improvement may be cheaper than random data access).

Reply:
800ms sounds good! For some reason I thought we measure ref time in nanoseconds, but ty @crystalin